src/master-playlist-controller.js

```diff
@@ -523,21 +523,30 @@ export class MasterPlaylistController extends videojs.EventTarget {
    */
   fastQualityChange_() {
     const media = this.selectPlaylist();
+    const isPaused = this.tech_.paused();
 
     if (media === this.masterPlaylistLoader_.media()) {
       return;
     }
 
+    this.tech_.pause();
```

Comment:
Is it necessary for us to pause? I think that may have inadvertent side effects.

Reply:
The rendition switch occurs without a stutter even without pausing/playing, but for some reason `timeupdate` events continue to fire even while playback has stopped and the loading spinner isn't shown. This only happens in IE/Edge.